lib/parsers/gjs-gts-parser.js

```diff
@@ -260,6 +261,34 @@ function preprocessGlimmerTemplates(info, code) {
         n.loc.start = codeLines.offsetToPosition(tpl.templateRange[0]);
         n.loc.end = codeLines.offsetToPosition(tpl.templateRange[1]);
       }
+      // split up element node into sub nodes to be able to reference tag name
+      // parts <Foo.Bar /> -> nodes for `Foo` and `Bar`
+      if (n.type === 'ElementNode') {
```

Comment:
Shouldn't this be @glimmer/syntax responsibility?

Reply:
i think it should, yes.
same for block params, there is no node for them

Reply:
Excellent, can you open an issue over on glimmer-vm?

Once we get that shipped, we can clean up here